BookingsApi/BookingsApi/Services/FeatureFlagService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BookingsApi.Contract.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace BookingsApi.Services
+{
+    public class FeatureFlagService : IFeatureFlagService
+    {
+        private readonly FeatureFlagConfiguration _featureFlagConfigurationOptions;
+        public FeatureFlagService(IOptions<FeatureFlagConfiguration> featureFlagConfigurationOptions)
+        {
+            _featureFlagConfigurationOptions = featureFlagConfigurationOptions.Value;
+        }
+
+        public bool GetFeatureFlag(string featureName)
```

Comment:
Nice implementation. Wonder if we want to capture the exception/default case where the passed feature flag name doesn't exist and maybe raise an appropriate exception.